core/main.go

```diff
@@ -61,22 +74,42 @@ func deployCoreServices(c *config.Config, api *api.API) error {
 	return nil
 }
 
+func stopRunningServices(api *api.API) error {
+	services, err := api.ListServices()
+	if err != nil {
+		return err
+	}
+	for _, s := range services {
+		status, err := s.Status()
+		if err != nil {
+			return err
+		}
+		if status == service.STARTING || status == service.PARTIAL || status == service.RUNNING {
```

Comment:
This test is useless because it's done in `service.Stop()` already:
https://github.com/mesg-foundation/core/blob/348aa982644ff31e9571813b298919dd99ed1c85/service/stop.go#L10-L20

Reply:
done